perun-core/src/main/resources/hsqldbChangelog.txt

```diff
@@ -4,6 +4,8 @@
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
 -- this update is not supported on hsql since its used only as in-memory db
+3.1.40
+alter table tasks alter column engine_id type integer;
```

Comment:
DB version bump must be part of each version changelog. Please add
```sql
insert into configurations (property,value) values ('DATABASE VERSION','3.1.40');
```
to ``hsqldbChangelog.txt``, ``oracleChangelog.txt`` and ``postgresChangelog``.